System/Random.hs

```diff
@@ -219,10 +214,14 @@ import Foreign.C.Types
 import Foreign.Marshal.Alloc (alloca)
 import Foreign.Ptr (plusPtr)
 import Foreign.Storable (peekByteOff, pokeByteOff)
-import GHC.Exts (Ptr(..), build)
```

Comment:
Isn't `build` still used in `randomRs` and `randoms`?

Reply:
It is also exported from `GHC.Base`